akka-typed/src/main/scala/akka/typed/ActorRef.scala

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright (C) 2014 Typesafe Inc. <http://www.typesafe.com>
+ */
+package akka.typed
+
+import akka.actor.ActorPath
+import scala.annotation.unchecked.uncheckedVariance
+import language.implicitConversions
+
+/**
+ * An ActorRef is the identity or address of an Actor instance. It is valid
+ * only during the Actor’s lifetime and allows messages to be sent to that
+ * Actor instance. Sending a message to an Actor that has terminated before
+ * receiving the message will lead to that message being discarded; such
+ * messages are delivered to the [[akka.actor.DeadLetter]] channel of the
+ * [[akka.actor.ActorSystem!.eventStream EventStream]] on a best effort basis
+ * (i.e. this delivery is not reliable).
+ */
+abstract class ActorRef[-T] extends java.lang.Comparable[ActorRef[_]] { this: ScalaActorRef[T] ⇒
```

Comment:
Good point. I think it's about as easy to confuse as `akka.actor.ActorRef` and `akka.typed.ActorRef`, maybe even less, because the name is slightly different. But as there is no relationship in the types, the compiler will tell you, as opposed to e.g. `scala.collection.Seq` and `mutable` vs `immutable`. ;-)